crates/ln-dlc-node/src/fee_rate_estimator.rs

```diff
@@ -9,6 +9,14 @@ use std::sync::RwLock;
 use std::sync::RwLockReadGuard;
 use std::sync::RwLockWriteGuard;
 
+const CONFIRMATION_TARGETS: [(ConfirmationTarget, usize); 3] = [
+    // We choose an extremely high background confirmation target to avoid force-closing channels
+    // unnecessarily.
+    (ConfirmationTarget::Background, 1000),
```

Comment:
@luckysori why not `1008`? 😅

Reply:
I didn't get that it was meant to be 7 days!